packages/client/src/v2-events/features/events/useCertificateTemplateSelectorFieldConfig.test.ts

```diff
@@ -0,0 +1,211 @@
+      italics:
+        'http://localhost:3000/api/countryconfig/fonts/NotoSans-Regular.ttf',
+      bolditalics:
+        'http://localhost:3000/api/countryconfig/fonts/NotoSans-Regular.ttf'
+    }
+  },
+  hash: '355011756a1cef0be4a032961d0e87b16c01f495-gzip',
+  svg: ''
+}
+
+const deathCertificateTemplateWithNoConditional: CertificateTemplateConfig = {
+  id: 'death-certificate',
+  event: 'death',
+  label: {
+    id: 'certificates.death.certificate',
+    defaultMessage: 'Death Certificate',
+    description: 'The label for a death certificate'
+  },
+  isDefault: true,
+  fee: {
+    onTime: 3,
+    late: 5.7,
+    delayed: 12
+  },
+  svgUrl: '/api/countryconfig/certificates/death-certificate.svg',
+  fonts: {
+    'Noto Sans': {
+      normal:
+        'http://localhost:3000/api/countryconfig/fonts/NotoSans-Regular.ttf',
+      bold: 'http://localhost:3000/api/countryconfig/fonts/NotoSans-Bold.ttf',
+      italics:
+        'http://localhost:3000/api/countryconfig/fonts/NotoSans-Regular.ttf',
+      bolditalics:
+        'http://localhost:3000/api/countryconfig/fonts/NotoSans-Regular.ttf'
+    }
+  },
+  hash: 'bf292f852fe65806734c4168ab3c86c12a1a0168-gzip',
+  svg: ''
+}
+
+const dateToday = new Date()
+
+function birthCertificateTemplateWithMinimumAge(
+  minimumAge: number
+): CertificateTemplateConfig {
+  return {
+    ...birthCertificateTemplateWithNoConditional,
+    conditionals: [
+      {
+        type: 'SHOW',
+        conditional: {
+          type: 'object',
+          properties: {
+            $form: {
+              type: 'object',
+              properties: {
+                'child.dob': {
+                  type: 'string',
+                  format: 'date',
+                  formatMaximum: new Date(
+                    dateToday.getFullYear() - minimumAge,
+                    dateToday.getMonth(),
+                    dateToday.getDate()
+                  ).toISOString()
+                }
+              },
+              required: ['child.dob']
+            }
+          },
+          required: ['$form']
+        } as unknown as JSONSchema
+      }
+    ]
+  }
+}
+const declaration = {
+  'mother.firstname': 'Mom',
+  'mother.surname': 'Test',
+  'mother.dobUnknown': true,
+  'mother.age': '80',
+  'mother.nationality': 'FAR',
+  'mother.idType': 'NATIONAL_ID',
+  'mother.nid': '9898989898',
+  'mother.address': {
+    country: 'ZAF',
+    addressType: 'INTERNATIONAL',
+    state: 'Western Cape',
+    district2: 'City Centre',
+    cityOrTown: 'Cape Town'
+  },
+  'mother.maritalStatus': 'SINGLE',
+  'mother.educationalAttainment': 'NO_SCHOOLING',
+  'mother.occupation': '',
+  'father.detailsNotAvailable': true,
+  'father.reason': 'Ran',
+  'informant.relation': 'MOTHER',
+  'informant.phoneNo': '0791234567',
+  'informant.email': 'mom@example.com',
+  'child.firstname': 'Kido',
+  'child.surname': 'Test',
+  'child.gender': 'male',
+  'child.dob': dateToday.toISOString(),
```

Comment:
This is the only line in the declaration you actually need. You could delete every other field and nothing would change. It's a test so only set up data you actually need.